app/controllers/reports_controller.rb

```diff
@@ -10,11 +10,13 @@ class ReportsController < ApplicationController
   add_breadcrumb I18n.t("breadcrumbs.reports"), :reports_path
 
   def index
-    @reports = Report.filter_start_date(params[:start_date])
-                     .filter_end_date(params[:end_date])
-                     .filter_name_status(params[:name], params[:status])
-                     .by_recently_created
-    @pagy, @reports = pagy @reports, items: Settings.pagy.number_items_10
+    @all_reports = Report.filter_start_date(params[:start_date])
+                         .filter_end_date(params[:end_date])
+                         .filter_name_status(params[:name], params[:status])
+                         .by_recently_created
+    @pagy, @reports = pagy @all_reports, items: Settings.pagy.number_items_10
+
+    export_xlsx
```

Comment:
```suggestion
    respond
```
Tên method là `respond` thôi a, nó có phải chỉ mỗi export xlsx ko đâu, còn respond về html bình thường nữa